server.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+require('./data/weather.json')
+require('dotenv').config();
+const express =require('express');
+const cors = require('cors');
+const list = require('./data/weather.json');
+
+const app = express();
+
+app.use(cors());
+
+const PORT = process.env.PORT;
+
+
+
+
+
+app.get('/weather', handleGetWeather)
+
+
+
+
+
+
+function handleGetWeather(request, response) {
+  console.log(request)
+  console.log('request.query', request.query)
+  // const dataToReturn = list.map(cityDataObject => {
+  //   let lon = cityDataObject.lon;
+  //   let lat = cityDataObject.lat;
+  //   return lon, lat
+  // });
+  // response.status(200).send(dataToReturn)
+  const lat = list.map(cityDataObject => {return cityDataObject.lat});
+  const lon = list.map(cityDataObject => {return cityDataObject.lon});
+  // response.status(200).write(lon, lat);
+  response.status(200).send(lat);
```

Comment:
- The lab wants us to use the .find() method to discover which city the lat, lon and searchQuery information belong to. If the user did not search for one of the three cities that we have information about (Seattle, Paris, or Amman), return an error.
- In this file we will also create our `class Forecast` with the properties of `date` and `description`.

If you have questions I am happy to help! 👍